### PR TITLE
Add some feat

### PR DESCRIPTION
**Pull request by alizaya**

### CODE REVIEW

File: app/Http/Controllers/AdminFoodController.php [hunk 1]
```diff
@@ -52,4 +74,13 @@ public function addFoodSubmit(Request $request)
 
         return redirect()->back()->with('msg-ok', __('msg.add_ok', ['name' => $request->get('name')]));
     }
+
+    public function foodsList()
+    {
+        allowed(Role::FOOD_MANAGER);
+
+        $data['foods'] = Food::query()->orderBy('restaurant_id')->get();
```
Comment (MahdiMajidzadeh) [inline app/Http/Controllers/AdminFoodController.php]: please use Eager Loading

File: resources/views/admin_food/foods_list.blade.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <thead>
+        <tr>
+            <th scope="col">نام غذا</th>
+            <th scope="col">قیمت</th>
+            <th scope="col">رستوران</th>
+        </tr>
+        </thead>
+        <tbody>
+        @foreach($foods as $food)
+            <tr>
```
Comment (MahdiMajidzadeh) [inline resources/views/admin_food/foods_list.blade.php:17]: please add `$food->id` too

File: resources/views/admin_food/restaurants_list.blade.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@section('inner-content')
+    <table class="table table-striped">
+        <thead>
+        <tr>
+            <th scope="col">اسم</th>
+        </tr>
+        </thead>
+        <tbody>
+        @foreach($restaurants as $restaurant)
+            <tr>
```
Comment (MahdiMajidzadeh) [inline resources/views/admin_food/restaurants_list.blade.php:15]: please add `$restaurant->id`

File: app/Http/Controllers/AdminFoodController.php (hunk 1 above)
Comment (alizaya) [reply app/Http/Controllers/AdminFoodController.php:82]: there is no relation here ! for eager loading

File: resources/views/admin_food/foods_list.blade.php (hunk 2 above)
Comment (alizaya) [reply resources/views/admin_food/foods_list.blade.php:17]: ok

File: app/Http/Controllers/AdminFoodController.php (hunk 1 above)
Comment (alizaya) [reply app/Http/Controllers/AdminFoodController.php]: done

**Review by MahdiMajidzadeh — APPROVED** [review]

**Pull request merged**